### PR TITLE
integrating front with API

### PR DESCRIPTION
**Pull request by Mohammadhussain257**
Create: Layout for help desk
Integrate: Integrate front-end with API

### CODE REVIEW

File: src/app/@core/models/index.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { from } from 'rxjs';
```
Comment (elwyncrestha) [inline src/app/@core/models/index.ts:1]: Remove this import.

File: src/app/feature/modules/helpdesk/components/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { from } from 'rxjs';
```
Comment (elwyncrestha) [inline src/app/feature/modules/helpdesk/components/index.ts:1]: Remove this import.

File: src/app/feature/modules/helpdesk/help-desk.module.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { HelpDeskComponent } from './components/help-desk/help-desk.component';
+import { HelpDeskRoutingModule } from './help-desk-routing.module';
+import { ThemeModule } from 'src/app/@theme/theme.module';
+import { QueryListComponent } from './components/query-list/query-list.component';
+
+@NgModule({
+  declarations: [HelpDeskComponent, QueryListComponent],
+  imports: [CommonModule, HelpDeskRoutingModule, ThemeModule],
+  exports: [HelpDeskComponent],
+  entryComponents: [HelpDeskComponent],
```
Comment (elwyncrestha) [inline src/app/feature/modules/helpdesk/help-desk.module.ts:12]: Remove `HelpDeskComponent` from entryComponent array as it is not being used as an entry component.

**Comment by Mohammadhussain257** [comment]
unnecessary import has been removed please review.

**Pull request merged**